C4SGSvc/src/main/java/org/c4sg/constant/UserDisplay.java

```diff
@@ -1,9 +1,29 @@
 package org.c4sg.constant;
 
 public enum UserDisplay {
```

Comment:
Why so complex? Boolean displayUser isn`t enough?
No need to manipulate with strings, less code...